lib/std/special/compiler_rt/README.md

```diff
@@ -1,15 +1,258 @@
+- port llvm __lshrti3              // a >> b  logical    (zero fill)
+- port llvm __negdi2               // -a
+- port llvm __negti2               // -a
+- none none __mulsi3               // a * b  missing signed
+- port llvm __muldi3               // a * b  signed
+- port llvm __multi3               // a * b  signed
+- port llvm __divsi3               // a / b  signed
+- port llvm __divdi3               // a / b  signed
+- port llvm __divti3               // a / b  signed
+- port llvm __udivsi3              // a / b  unsigned
+- port llvm __udivdi3              // a / b  unsigned
+- port llvm __udivti3              // a / b  unsigned
+- port llvm __modsi3               // a % b  signed
+- port llvm __moddi3               // a % b  signed
+- port llvm __modti3               // a % b  signed
+- port llvm __umodsi3              // a % b  unsigned
+- port llvm __umoddi3              // a % b  unsigned
+- port llvm __umodti3              // a % b  unsigned
+- port llvm __udivmoddi4           // a / b, rem.* = a % b  unsigned
+- port llvm __udivmodti4           // a / b, rem.* = a % b  unsigned
+- port llvm __udivmodsi4           // a / b, rem.* = a % b  unsigned
+- port llvm __divmodsi4            // a / b, rem.* = a % b  signed, ARM
+
+####  Arithmetic with trapping overflow
+- dev BitTwiddlingHacks __absvsi2  // abs(a)
+- dev BitTwiddlingHacks __absvdi2  // abs(a)
+- dev BitTwiddlingHacks __absvti2  // abs(a)
+- port llvm __negvsi2              // -a
+- port llvm __negvdi2              // -a
+- port llvm __negvti2              // -a
+- TODO upstreaming __addvsi3..__mulvti3 after testing panics works
+- dev HackersDelight __addvsi3     // a + b
+- dev HackersDelight __addvdi3     // a + b
+- dev HackersDelight __addvti3     // a + b
+- dev HackersDelight __subvsi3     // a - b
+- dev HackersDelight __subvdi3     // a - b
+- dev HackersDelight __subvti3     // a - b
+- dev HackersDelight __mulvsi3     // a * b
+- dev HackersDelight __mulvdi3     // a * b
+- dev HackersDelight __mulvti3     // a * b
+
+#### Arithmetic which returns if overflow (would be faster without pointer)
+- dev HackersDelight __addosi4     // a * b, overflow=>ov.*=1 else 0
+- dev HackersDelight __addodi4     // a * b, overflow=>ov.*=1 else 0
+- dev HackersDelight __addoti4     // a * b, overflow=>ov.*=1 else 0
+- dev HackersDelight __subosi4     // a * b, overflow=>ov.*=1 else 0
+- dev HackersDelight __subodi4     // a * b, overflow=>ov.*=1 else 0
+- dev HackersDelight __suboti4     // a * b, overflow=>ov.*=1 else 0
+- dev HackersDelight __mulosi4     // a * b, overflow=>ov.*=1 else 0 (required by llvm)
+- dev HackersDelight __mulodi4     // a * b, overflow=>ov.*=1 else 0 (required by llvm)
+- dev HackersDelight __muloti4     // a * b, overflow=>ov.*=1 else 0 (required by llvm)
+
+## Soft float library routines
+
+#### Conversion
+
+- __extendsfdf2  // extend a f32 => f64
+- __extendsftf2  // extend a f32 => f128
+- __extendsfxf2  // extend a f32 => f80 missing
```

Comment:
This and `__truncxfsf2` exist but I misnamed them using `f` instead of `s`.

Reply:
renamed.